infrastructure/src/main/java/org/corfudb/infrastructure/ManagementAgent.java

```diff
@@ -105,6 +113,43 @@
     @Getter
     private volatile CompletableFuture<Boolean> sequencerBootstrappedFuture;
 
+    /**
+     * Timeout before which the management agent attempts to bootstrap a NOT_READY sequencer.
+     */
+    private static final int sequencerStateHeartBeatCounter = 3;
+
+    //    private Map.Entry<Long, Integer> seqNotReadyCountAtEpoch;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)